src/pages/bookmarks/Bookmarks.tsx

```diff
@@ -1,5 +1,72 @@
-import { FC } from "react";
+import { PostCard } from "components";
+import { FC, useEffect } from "react";
+import { Grid as Loader } from "react-loader-spinner";
+import { useSelector } from "react-redux";
+import { useDispatch } from "react-redux";
+import { AppDispatch, RootState } from "store";
+import "./bookmarks.css";
+import { useAxios } from "utils";
+import { updateBookmarks } from "reducers/postsSlice";
+import Lottie from "react-lottie";
+import animationData from "lotties/not-found.json";
+
+const defaultOptions = {
+  loop: true,
+  autoplay: true,
+  animationData: animationData,
+  rendererSettings: {
+    preserveAspectRatio: "xMidYMid slice",
+  },
+};
 
 export const Bookmarks: FC = () => {
```

Comment:
Appreciation : Use of animations is great. Lottie animations are looking good , but with the loader its getting blurred out

Reply:
Thanks!